docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1d.md

```diff
@@ -0,0 +1,82 @@
+# does_zone_meet_G3_1_1d  
+
+**Schema Version:** 0.0.28
+
+**Description:** determines whether a given zone meets the G3_1_1d exception "For laboratory spaces in a building having a total laboratory 
+exhaust rate greater than 15,000 cfm, use a single system of type 5 or 7 serving only those spaces.  The lab exhaust fan shall be modeled 
+as constant horsepower reflecting constantvolume stack discharge with outdoor air bypass."
```

Comment:
I don't think this function checks for how lab exhaust is modeled, so please remove the second sentense in the Description (The lab exhaust fan shall be modeled as constant horsepower reflecting constantvolume stack discharge with outdoor air bypass.") and make it a separate RDS.

Reply:
updated